lua/core/pack.lua

```diff
@@ -25,11 +26,16 @@ function Lazy:load_plugins()
 	local append_nativertp = function()
 		package.path = package.path
 			.. string.format(";%s;%s", modules_dir .. "/configs/?.lua", modules_dir .. "/configs/?/init.lua")
+			.. string.format(";%s;%s", user_modules_dir .. "/configs/?.lua", user_modules_dir .. "/configs/?/init.lua")
 	end
 
 	local get_plugins_list = function()
 		local list = {}
 		local plugins_list = vim.split(fn.glob(modules_dir .. "/plugins/*.lua"), "\n")
+		local user_plugins_list = vim.split(fn.glob(user_modules_dir .. "/plugins/*.lua"), "\n")
+		if user_plugins_list[1] ~= "" then
```

Comment:
```suggestion
		if not vim.tbl_isempty(user_plugins_list) then
```
Better to use native Lua API's to cope with possible corner cases.